config/nginx.conf.development.template

```diff
@@ -151,6 +151,12 @@ http {
       proxy_pass http://planet:9876;
     }
 
+    location /studio/ {
```

Comment:
I can't remember the rules about the trailing slash here, @AmasiaNalbandian or @manekenpix might remember.